masterlist.yaml

```
@@ -3082,6 +3082,24 @@ plugins:
       - crc: 0xE5BCCA66
         util: 'TES4Edit v4.0.4'
 
+  - name: 'Magic Visuals Overhaul.esp'
+    url: [ 'https://www.nexusmods.com/oblivion/mods/49300/' ]
+    req: [ *OBSE ]
+    msg:
+      - <<: *obsolete
+        subs: [ '[Lightning Crash Fix](https://www.nexusmods.com/oblivion/mods/49300/)' ]
+        condition: 'checksum("Magic Visuals Overhaul.esp", EC596124)'
+      - <<: *patch3rdParty
+        type: warn
+        subs:
+          - 'Oblivion Magic Extender'
+          - '[Magic Visuals Overhaul - OBME Patch](https://www.nexusmods.com/oblivion/mods/49096/)'
```

The message refers to a mod page where the patch is hosted, so you should use the mod page's name here. However, you can specify which file you're referring to.

```suggestion
          - '[Av Latta Magicka - Oblivion Magic Overhaul](https://www.nexusmods.com/oblivion/mods/49096/) (Magic Visuals Overhaul - OBME Patch)'
```